### ISSUE TITLE
Pro Audio Spectrum Sound bug

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#4334]
### What happened?

I tested Prince of Persia 1 v1.3 again. It works fine if I begin the game directly, but there is a bug during the game presentation when Jaffar opens the door.

Here is the video capture of the game. (Jaffar opens the door at 1:02)

https://github.com/86Box/86Box/assets/31600563/82e7ce69-abeb-41b9-8cfa-ae8ce6cfe98d

Here is the VM,CFG and HDD imae files:
[386 DX multimedia + PAS Plus.zip](https://github.com/86Box/86Box/files/14891396/386.DX.multimedia.%2B.PAS.Plus.zip)

And the PoP 1 floppy disk:
[PRINCE OF PERSIA 1.3.zip](https://github.com/86Box/86Box/files/14891404/PRINCE.OF.PERSIA.1.3.zip)

### Configuration file

```ini
[Tseng Labs ET4000AX (TC6058AF) (ISA)]
memory = 1024
bios_ver = v1_10

[Microsoft Serial Mouse]
port = 0
buttons = 2

[Pro Audio Spectrum Plus]
receive_input401 = 0
receive_input = 1

[General]
vid_renderer = qt_software
vid_resize = 1
rctrl_is_lalt = 1
enable_overscan = 1
host_cpu = Intel(R) Core(TM) i5-4210U CPU @ 1.70GHz
emu_build_num = 5678
uuid = ec5c3b85-4224-5daf-80bc-b44119e59882

[Machine]
machine = asus386
cpu_family = i386dx
cpu_speed = 25000000
cpu_multi = 1
cpu_use_dynarec = 0
fpu_softfloat = 0
time_sync = local
pit_mode = 1
mem_size = 4096

[Video]
gfxcard = et4000ax

[Input devices]
mouse_type = msserial

[Sound]
sndcard = pasplus
fm_driver = nuked
sound_type = int16

[Network]
net_01_link = 0
net_02_link = 0
net_03_link = 0
net_04_link = 0

[Ports (COM & LPT)]
lpt1_device = text_prt

[Storage controllers]
hdc = ide_isa
cassette_mode = load

[Hard disks]
hdd_01_parameters = 17, 15, 900, 0, ide
hdd_01_speed = 1992_3600rpm
hdd_01_vhd_blocksize = 4096
hdd_01_ide_channel = 0:0

[Floppy and CD-ROM drives]
fdd_01_type = 35_2hd
fdd_02_type = 525_2hd
cdrom_01_speed = 4
cdrom_01_parameters = 1, atapi
cdrom_01_ide_channel = 0:1
cdrom_01_host_drive = 200
cdrom_01_type = AZT_CDA46802I_1.15


[Tseng Labs ET4000AX (ISA)]
memory = 1024
bios_ver = v8_06

[Trident TVGA 8900D-R]
memory = 1024

[Pro Audio Spectrum 16]
receive_input401 = 0
receive_input = 1
```

### Operating system

Windows 10 22H2

### CPU

i5 4210U

### 86Box version

4.2.0.5678

### Build architecture

Windows - x86 (32-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

Official website (Jenkins, GitHub)

**Comment by TC1995** [comment]
Maybe an IRQ/DMA conflict?

**Comment by TC1995** [comment]
Wait, I think the drivers you're using aren't for the PAS Plus.

**Comment by Fenix770** [comment]
I think it should be backward compatible with the PAS-8, unless the sound effect use some feature of the OPL-2 chip of the  8-bit card and no present on PAS-PLUS. The only difference I found between Plus and 16, is the 16 have the SB wavetable and can emulate the SB.

Also, I found that if I enable the SB emulation with MVSOUND.SYS file. I get an error on the PAS Plus when loading Win 3.0 ME, but no on PAS-16. (The emulation is enabled with /S:1,200,1,5 parameter)
I have never seen a real PAS Plus but maybe is not SB compatible, at least not as the PAS-16 is.

![Monitor_1_20240405-220857-368](https://github.com/86Box/86Box/assets/31600563/04ce80d6-660b-4a79-87e4-fd46a13ee7e2)
